app/src/main/java/com/pavesid/androidacademy/utils/Utils.kt

```diff
@@ -1,42 +1,20 @@
 package com.pavesid.androidacademy.utils
 
-import android.content.Context
-import android.graphics.Point
-import android.os.Build
-import android.view.WindowManager
+import android.content.res.Resources
+import android.util.TypedValue
+import androidx.annotation.AttrRes
+import androidx.annotation.ColorInt
 
 object Utils {
 
-    fun getNavigationBarSize(context: Context): Point? {
-        val appUsableSize: Point = getAppUsableScreenSize(context)
-        val realScreenSize: Point = getRealScreenSize(context)
-
-        return Point(realScreenSize.x - appUsableSize.x, realScreenSize.y - appUsableSize.y)
-    }
-
-    private fun getAppUsableScreenSize(context: Context): Point {
-        val windowManager =
-            context.getSystemService(Context.WINDOW_SERVICE) as WindowManager
-        val display = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
-            context.display
-        } else {
-            windowManager.defaultDisplay
-        }
-        val size = Point()
-        display?.getSize(size)
-        return size
-    }
-
-    private fun getRealScreenSize(context: Context): Point {
-        val windowManager =
-            context.getSystemService(Context.WINDOW_SERVICE) as WindowManager
-        val display = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
-            context.display
-        } else {
-            windowManager.defaultDisplay
-        }
-        val size = Point()
-        display?.getRealSize(size)
-        return size
+    @ColorInt
+    fun getColorFromAttr(
```

Comment:
usually it is not kotlin way to create Utils object.
Better practice is to create file like `ResoursesExt.kt` or `ThemeExt.kt` and all functions place in file without class or object.
So you function can be like 
```
fun Theme.getColorFromAttr(
     @AttrRes attrColor: Int,  
     typedValue: TypedValue = TypedValue(),
     resolveRefs: Boolean = true
)
```